node/lib/util/repo_ast_test_util.js

```diff
@@ -230,6 +239,12 @@ exports.testMultiRepoManipulator =
     else {
         assert.isFunction(options.expectedTransformer);
     }
+    if (!("actualTransformer" in options)) {
```

Comment:
nit: swap if/else clauses and make if statement positive rather than negative.

Reply:
I entered an issue to remind myself to fix this: https://github.com/twosigma/git-meta/issues/389